src/checkboxQuickPick.ts

```diff
@@ -0,0 +1,92 @@
+/*---------------------------------------------------------
+ * Copyright (C) Microsoft Corporation. All rights reserved.
+ *--------------------------------------------------------*/
+
+import vscode = require("vscode");
+import QuickPickItem = vscode.QuickPickItem;
+const figures: any = require("figures");
+
+export interface ICheckboxOption {
+    name: string;
+    isSelected: boolean;
+}
+
+export class CheckboxQuickPick {
+    private options: ICheckboxOption[];
+    private readonly confirm: string;
+    private readonly checkboxOn: string;
+    private readonly checkboxOff: string;
+    private readonly confirmPlaceHolder: string;
+
+    constructor(options: ICheckboxOption[]) {
+        this.options = options;
+        this.confirm = figures.tick;
```

Comment:
Instead of using the 'figures' package, it'd be more convenient to use the built-in icons that you can reference by a special string syntax.  Here's an example: https://github.com/PowerShell/vscode-powershell/blob/master/src/session.ts#L391.  Those string names are refer to the names of GitHub's [Octicons](https://octicons.github.com/).  In this case you could just use [`$(check)`](https://octicons.github.com/icon/check/) to achieve the same effect.